bedrock/newsletter/templates/newsletter/opt-out-confirmation.html

```diff
@@ -66,7 +66,7 @@ <h2>{{ ftl('opt-out-confirmation-prefer-to-get') }}</h2>
         <li><a class="instagram" href="{{ mozilla_instagram_url() }}" data-link-type="footer" data-link-text="Instagram (@mozilla)">{{ ftl('opt-out-confirmation-instagram') }}<span> (@mozilla)</span></a></li>
         <li><a class="youtube" href="https://www.youtube.com/firefoxchannel" data-link-type="footer" data-link-text="YouTube (firefoxchannel)">{{ ftl('opt-out-confirmation-youtube') }}<span> (firefoxchannel)</span></a></li>
         <li><a class="facebook" href="https://www.facebook.com/Firefox" data-link-type="footer" data-link-text="Facebook (Firefox)">{{ ftl('opt-out-confirmation-facebook') }}<span> (Firefox)</span></a></li>
-        <li><a class="twitter" href="{{ firefox_twitter_url() }}" data-link-type="footer" data-link-text="Twitter (@firefox)">{{ ftl('opt-out-confirmation-twitter') }}<span> (@firefox)</span></a></li>
+        <li><a class="twitter" href="{{ firefox_twitter_url() }}" data-link-type="footer" data-link-text="Twitter (@firefox)" aria-label="{{ ftl('opt-out-confirmation-x-formerly-twitter') }}">{{ ftl('opt-out-confirmation-x', fallback='opt-out-confirmation-twitter') }}<span> (@firefox)</span></a></li>
```

Comment:
The text is hidden here like other footer links, so dropping `aria-label` and just using `<a>X (formerly Twitter) @firefox</a>` is probably fine?